packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -207,7 +212,9 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
       if (exitCode === 98) {
         message = nls.localize('force_lightning_lwc_start_addr_in_use');
       }
-
+      if (exitCode === 135){
+        message = nls.localize('force_lightning_lwc_start_no_lwc_folder', 'testPath');
```

Comment:
I think we're good to remove the testPath portion. A customer should see the error message in the terminal if they need more information about the path.

```suggestion
        message = nls.localize('force_lightning_lwc_start_no_lwc_folder');
```